### ISSUE TITLE
Model responds that it cannot create and modify computer files

### ISSUE THREAD

**Issue by LaoJiuYes** [KillianLucas/open-interpreter#723]
### Describe the bug

When I use it and ask it to write code for me, it always replies to me that it "can't create a file on my computer".
Am I using it incorrectly?
I am having the above problem with both systems, windows 10 and ubuntu 22.4.
I use gpt-4.

### Reproduce

You just tell it to write a program that implements a certain function.

### Expected behavior

I read the blurb describing it as so smart, and I asked it about it when I started using it, and it said very much so itself, but rejected me many times when it came to using it specifically.
I wonder if I'm not using it the right way. Please guide me, thanks a million.

### Screenshots

![QQ截图20231101161037](https://github.com/KillianLucas/open-interpreter/assets/53503340/b7c46e20-e920-45bd-8230-2f706e172162)

### Open Interpreter version

0.1.10

### Python version

3.11.5

### Operating System name and version

Windows 10/Ubuntu 22.4

**Comment by ericrallen** [comment]
Hey there, @LaoJiuYes!

There have been some reports of a similar problem with the GPT-3.5 Turbo model, but I haven’t seen this with GPT-4 yet.

Could you provide some more context to help us narrow it down?

Have you modified the `system_measage`?

Are you using OpenAI models directly or through a proxy or another service?

**Comment by vatogato** [comment]
I have a feeling it has to do with custom instructions and the new multimodal functionality maybe.  I encountered similar yesterday with prompts I'd been using before the changes, and the outputs were different, slower, and I actually got the cannot execute code message, which was funny because it was all about linguistic exploration, not code execution at all.

I removed my custom instructions and it helped, but didn't solve it completely. 

I continued my investigation using the OI system prompt below:

with custom instructions "precisely."
https://chat.openai.com/share/f24a3d0f-8b58-46c5-b65f-5a72a28d4027

without custom instructions:
https://chat.openai.com/share/02e459b9-4812-4703-b8da-2101f29175ce
